Client/src/Pages/Settings/index.jsx

```diff
@@ -300,15 +320,16 @@ const Settings = ({ isAdmin }) => {
           </Box>
           <Box>
             <Typography component="h2">BlueWave Uptime v1.0.0</Typography>
+            <Typography component="h2">Latest Version: {version}</Typography>
```

Comment:
We don't need both of these, you can just combine them to

`BlueWave Uptime: {version}`